src/TaskItem.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+const TaskItem = ({ task, toggleCompleted }) => {
+    const {id, name, due, completed} = task;
```

Comment:
I tried structuring this object line by line for better readability, but it looks kinda weird?
```
const {
    id,
    name,
    due,
    completed
} = task;
```

Reply:
This is fine. You don't need to do it line by line. You can start looking into ESLint and prettier. It will auto formatted for you.